be/src/runtime/stream_load/stream_load_executor.cpp

```diff
@@ -148,6 +148,13 @@ Status StreamLoadExecutor::execute_plan_fragment(std::shared_ptr<StreamLoadConte
                 static_cast<void>(this->commit_txn(ctx.get()));
             }
         }
+
+        LOG(INFO) << "finished to execute job. label=" << ctx->label << ", txn_id=" << ctx->txn_id
```

Comment:
leave a keyword such as streamload in the log. 
and consider adding a bvar latency recorder for streamload

Reply:
> leave a keyword such as streamload in the log.

I will update it.

> and consider adding a bvar latency recorder for streamload

Perhaps bvar needs to be on-site, but without it, we may not know what happened?